contrib/babelfishpg_common/src/typecode.c

```diff
@@ -17,6 +17,12 @@
 /*  Memory context  */
 MemoryContext TransMemoryContext = NULL;
 
+/* 
+   Note: For performance optimization there is a view sys.type_info_internal 
+   defined in contrib/babelfishpg_tsql/sql/sys_views.sql file.
+   That view contains a 'map' of the psql to sql server names specified here.
```

Comment:
avoid `sql server` term in our code.

Reply:
Removed the whole comment, as it does not apply any more since I removed the sys.type_info_internal view.